step.go

```diff
@@ -1,30 +1,49 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
+	"strings"
+	"time"
+
 	"github.com/bitrise-io/go-steputils/stepconf"
 	"github.com/bitrise-io/go-utils/command"
 	"github.com/bitrise-io/go-utils/env"
 	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/pathutil"
+	"github.com/bitrise-steplib/bitrise-step-artifact-pull/api"
+	"github.com/bitrise-steplib/bitrise-step-artifact-pull/downloader"
+	"github.com/bitrise-steplib/bitrise-step-artifact-pull/model"
 )
 
+const downloadDirPrefix = "_artifact_pull"
+
 type Input struct {
-	Example string `env:"example,required"`
+	Verbose               bool            `env:"verbose,required"`
```

Comment:
If a field is "value options" you can list options so we validate the value while parsing. Example: https://github.com/bitrise-steplib/steps-apt-get-install/blob/c6e84238d4fe5afe369293893a8094a5e8154cbe/main.go#L19